internal/martian/httpspec/httpspec.go

```diff
@@ -33,6 +33,7 @@ func NewStack(via string) (outer, inner *fifo.Group) {
 	outer.AddRequestModifier(hbhm)
 	outer.AddRequestModifier(header.NewForwardedModifier())
 	outer.AddRequestModifier(header.NewBadFramingModifier())
+	outer.AddRequestModifier(header.NewRequestIDModifier())
```

Comment:
This is not part of `httpspec`!

Actually this should be controlled by a flag and maybe users should have control over name because adding X-Request-ID may cause some unexpected behavior.

Reply:
You're right, it's not part of the spec. It was comfortable to put it here.

Reply:
I will add a flag.